OpenSim/Examples/OptimizationExample_Arm26/OptimizationExample.cpp

```diff
@@ -160,7 +160,7 @@ int main()
         //Optimizer opt(sys, InteriorPoint);
 
         // Specify settings for the optimizer
-        opt.setConvergenceTolerance(0.1);
+        opt.setConvergenceTolerance(0.11);
```

Comment:
>Curious change. Was it was necessary to get convergence? Why would that be?

Running the test without this changes takes over 10 minutes (which I think is too long) and you eventually get the following exception:
```
...
objective evaluation #: 375  controls = ~[0.01 0.01 0.01 0.39741 1.01133 0.353332] bestSoFar = -5.00901
objective evaluation #: 379  controls = ~[0.01 0.01 0.01 0.425754 0.99 0.391037] bestSoFar = -5.0228
objective evaluation #: 389  controls = ~[0.01 0.01 0.01 0.425754 1.01133 0.391037] bestSoFar = -5.03798
objective evaluation #: 393  controls = ~[0.01 0.01 0.01 0.539129 0.99 0.541859] bestSoFar = -5.13268
objective evaluation #: 403  controls = ~[0.01 0.01 0.01 0.539129 1.01133 0.541859] bestSoFar = -5.14832
objective evaluation #: 416  controls = ~[0.01 0.01 0.01 0.889218 0.99 0.99] bestSoFar = -5.19584
objective evaluation #: 423  controls = ~[0.0121544 0.01 0.01 0.698804 0.99 0.735429] bestSoFar = -5.27781
objective evaluation #: 523  controls = ~[0.01 0.013971 0.0101359 0.901013 0.99 0.99] bestSoFar = -5.40065
SimTK Exception thrown at LBFGSBOptimizer.cpp:119:
  Optimizer failed: ABNORMAL_TERMINATION_IN_LNSRCH
```